app/views/components/DateErrorSummary.scala.html

```diff
@@ -0,0 +1,29 @@
+@*
```

Comment:
I think Bonnie has added this file in her work as she needs it for the change to the date input page. I can't see where it's used at the moment though - can you remove it or am I missing something? 🙂